tests/contract_testcases/solana/issue732.sol

```diff
@@ -0,0 +1,20 @@
+library FixedPoint {
```

Comment:
How about specifying what you are trying to test in the file's name?
Nobody will know what `issue732.sol` tests just by looking at the code. You can mention in the file that it is related to the issue.

Reply:
@seanyoung

Reply:
I've renamed the file